traffic_ops/app/lib/API/User.pm

```diff
@@ -98,6 +98,7 @@ sub index {
 				"stateOrProvince" => $row->state_or_province,
 				"uid"             => $row->uid,
 				"username"        => $row->username,
+				"tenant"          => defined ($row->tenant) ? $row->tenant->name : "N/A",
```

Comment:
Can you just leave this undef if not defined? In other API's that's how it is done. I'd like to keep it consistent. So

"tenant"          => defined ($row->tenant) ? $row->tenant->name : undef,